src/multiply/view/main.py

```diff
@@ -75,6 +75,10 @@ def view(result_dir, genome_name):
         )
         primer_plotter = PrimerPlotter(target_primer_df)
 
+        # if there are no features to plot, skip this site
+        if gff_plotter.plot_gff.shape[0] == 0:
```

Comment:
It might be helpful to raise a warning here? e.g.
`warnings.warn(f"No features were found for plotting inside the region: {self.chrom}:{self.start}-{self.end}"). `